src/features/globalReadings/components/ListItem.tsx

```diff
@@ -109,8 +109,15 @@ const ListItem: React.FunctionComponent<ListItemProps> = ({
             <img
               loading="lazy"
               src={newReadingImage}
+              onError={(e) => {
```

Comment:
Can you move this anonymous function inside `imageOnErrorHandler()` and then move that function to be within `ListItem` below `getImageDimensions()`?